test/integration/localnet/get-version.test.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * LocalNet Integration Test for GetVersion API
+ *
+ * This test validates connectivity to a running LocalNet instance by calling the getVersion API and validating the full
+ * response structure.
+ *
+ * Prerequisites:
+ *
+ * - LocalNet must be running (set up by CircleCI or manually via scripts)
+ *
+ * This test verifies:
+ *
+ * - SDK can connect to LocalNet JSON API
+ * - GetVersion API returns the expected response
+ */
```

Comment:
Most (or all) of this comment can be removed. If anything it belongs in the setup.ts to be mentioned just once, not per test like this.